packages/expo-updates/android/src/main/java/expo/modules/updates/loader/Manifest.java

```diff
@@ -0,0 +1,179 @@
+package expo.modules.updates.loader;
+
+import android.net.Uri;
+import android.util.Log;
+
+import expo.modules.updates.UpdatesController;
+import expo.modules.updates.db.entity.AssetEntity;
+import expo.modules.updates.db.entity.UpdateEntity;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+import java.util.UUID;
+
+import static expo.modules.updates.loader.EmbeddedLoader.BUNDLE_FILENAME;
+
+public class Manifest {
+
+  private static String TAG = Manifest.class.getSimpleName();
+
+  private static String EXPO_ASSETS_URL_BASE = "https://d1wp6m56sqw74a.cloudfront.net/~assets/";
+  private static String[] EXPO_DOMAINS = new String[] {"expo.io", "exp.host", "expo.test"};
```

Comment:
Even at this early stage I think we should split this Expo-proprietary code out, or at least figure out a migration path forward so that newer versions of this module don't need this code.